Not sure if this is real or not, but it did happen in two different jobs of the build (the other was a HOPE api test):

```
======================================================================
FAIL: corehq.apps.api.tests:TestCommCareCaseResource.test_get_list
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/mnt/commcare-hq-ro/corehq/apps/api/tests.py", line 427, in test_get_list
    self.assertEqual(response.status_code, 200)
AssertionError: 400 != 200
```

--- a/corehq/apps/api/resources/v0_4.py
+++ b/corehq/apps/api/resources/v0_4.py
@@ -249,7 +249,7 @@ class CommCareCaseResource(SimpleSortableResourceMixin, v0_3.CommCareCaseResourc
     date_modified = fields.CharField(attribute='modified_on', default="1900-01-01")
     server_date_modified = fields.CharField(attribute='server_modified_on', default="1900-01-01")
     server_date_opened = fields.CharField(attribute='server_opened_on', default="1900-01-01")
-    opened_by = fields.CharField(attribute='opened_by')
+    opened_by = fields.CharField(attribute='opened_by', null=True)
 
     def obj_get(self, bundle, **kwargs):
         case_id = kwargs['pk']

